api/db/run-migrations.sh

```diff
@@ -2,15 +2,9 @@
 set -xeuo pipefail
 IFS=$'\n\t'
 
-activity="-PrunList=$1"
-if [ -z ${2+x} ]
-then
-    context=""
-else
-    context="-Pcontexts=$2"
-fi
-
-# Ruby is not installed in our dev container and this script is short, so bash is fine.
+liquibaseCommand="${1-update}"
+runList="-PrunList=${2-main}"
```

Comment:
We actually don't need to provide this explicitly, since the only activity we have is `main`. That would simplify things considerably, though I don't want to sacrifice generality. It would be more robust to set `main` as the default as well, if that's supported.